app/queries/user_conversations_query.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class UserConversationsQuery
+  attr_reader :relation, :user_id
+
+  def initialize(user_id, relation = Conversation.all)
+    @relation = relation
+    @user_id = user_id
+  end
+
+  def call
+    relation.where('sender_id = ? OR receiver_id = ?', user_id, user_id)
+  end
+end
```

Comment:
We can also add a `.between(necessary_args)` method and transfer the query in `ConversationBetweenUsersQuery.call` here. So we can have all the queries for model Conversation in one place.